acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -186,29 +186,55 @@ void ethSignerDoesNotConnectToServerNotSpecifiedInTrustStore(@TempDir Path workD
   @Test
   void missingKeyStoreForEthSignerResultsInEthSignerTerminating(@TempDir Path workDir)
```

Comment:
Test name isn't quite right now as EthSigner doesn't terminate. Maybe "missingKeyStoreForEthSignerResultsInRespondingInInternalServerError" or something better?

Reply:
yes, I noticed it as well after pushing the PR :winced:

Reply:
done